snapcraft/commands/microcloud

```diff
@@ -7,4 +7,14 @@ if [ -x "${SNAP_COMMON}/microcloud.debug" ]; then
     mkdir -p "${GOCOVERDIR}"
 fi
 
+# Change the location of $VIMRUNTIME root to be able to define the defaults.vim file.
+# This is to prevent seeing the message 'E1187: Failed to source defaults.vim'
+# which is caused by not having the 'vim-runtime' package.
+# Also see https://bugs.launchpad.net/ubuntu/+source/vim/+bug/1968912.
+export VIMRUNTIME="/snap/microcloud/current/usr/share/vim/vim91"
+export EDITOR="vim.tiny -Z --clean"
```

Comment:
In the LXD case, `snapcraft/wrappers/editor` was also dealing with `VIMINIT` to point `vim.tiny` to the `/etc/vimrc` file.

Reply:
I tried, but same outcome as in https://github.com/canonical/microcloud-pkg-snap/pull/33#discussion_r1852199042, the `defaults.vim` is still probed.